### DISCUSSION TITLE
Sobre ChromeApps

### DISCUSSION THREAD

**Discussion by ortizvinicius** [frontendbr/forum#1038] [Perguntas e Respostas]
Bom dia pessoal,

Estou com uma necessidade de desenvolver um sistema web que funcione offline (conexão zero mesmo), sincronizando dados online quando houver conexão, inicialmente apenas desktop mesmo.

Dei uma pesquisada nesses apps do Chrome, pelo que vi são web apps comuns com algumas APIs que o Chrome fornece, que a propósito achei interessante, por exemplo, a API de arquivos que resolve meu problema offline, além de outras legais como bluetooth, In-App Purchases e info. de hardware.

Também sei que com o próprio localStorage do JS consigo salvar informações offline.

Outra coisa que me chamou a atenção nos ChromeApps é a possibilidade de portar para mobile e publicar em App Stores.

Queria saber a opinião de vocês, alguém já desenvolveu um Chrome App? Este é o caminho certo pra mim? Existe alguma alternativa melhor? Alguma ressalva que devo saber?

**Comment by willycamargo** [comment]
Interessante, @ortizvinicius !

Apesar de ter lido sobre, não desenvolvi nada. Eu particularmente avaliaria a possibilidade de utilizar o [Electron](https://electronjs.org/) para uma solução desse tipo. Deixando o app "desvinculado" do chrome em sí.

**Comment by ortizvinicius** [comment]
@willycamargo Fala meu brother!
Cara sensacional, acho que a única coisa que perco é sistema de pagamentos do Chrome (na verdade ainda não sei se vou precisar e dá pra ser substituído de boa), mas agora estou mais inclinado a usar o Electron, ele me atende com certeza.

**Comment by felipefialho** [comment]
Electron é muito bacana mesmo, não consigo pensar em nada melhor para esse caso.
